backend/app/db/crud/document.py

```diff
@@ -47,3 +48,14 @@ def is_document_exists(
             )
         )
     ).scalar()
+
+
+def get_document(db: Session, document_id: int) -> Document:
+    document = db.query(Document).filter(Document.id == document_id).first()
```

Comment:
Also filter by `is_valid`?

Reply:
Ah okay, I'll add that - I thought that invalid documents weren't pushed into the api db

Reply:
Ah, maybe you're right, and there's nothing to do.